README.md

```diff
@@ -7,13 +7,13 @@ API spec available at https://pve.proxmox.com/pve-docs/api-viewer/index.html
 
 ## Requirements: ##
 
-PHP 5 with cURL (including SSL) support.
+PHP 7 with cURL (including SSL/TLS) support.
```

Comment:
Does this no longer work on 5.x? I'd feel better saying "5 or higher" if it isn't actually broken there. I know 5 is massively outdated, but I'm sure it's still in the wild somewhere.